### ISSUE TITLE
[MU4 Issue] [Accessibility] Options missing from context menu when triggered via keyboard

### ISSUE THREAD

**Issue by shoogle** [musescore/MuseScore#15379]
**Describe the bug**
If you right-click on a measure with the mouse then various options are provided in the context menu. However, many of these options are not provided if you select a measure with the keyboard then then simulate a right click by pressing one of these keys:

- `Menu` key (next to the right `Ctrl` key on most Windows keyboards)
- `Shift+F10` or `Ctrl+Shift+F10` (we provide these alternative shortcuts because not all laptops have a dedicated `Menu` key).

**To Reproduce**
1. Create an blank score.
2. Right-click on the staff lines in a measure (somewhere that is not a note, rest, or other element).
3. Notice the options available, then dismiss the menu.
5. Use `Shift` and the arrow keys to create a range selection of one or more measures.
6. Press the `Menu` key or one of the alternative shortcuts above (e.g. `Shift+F10`).
7. Notice that fewer options are now available in the context menu.

**Expected behavior**
The context menu for range selections should include the same options regardless of how the selection is made, and regardless of how the context menu is triggered (i.e. via mouse or via keyboard).

**Screenshots**

Right-click context menu | Keyboard context menu
---|---
![image](https://user-images.githubusercontent.com/11011881/208426395-ca614534-54c3-48f6-9509-b1c00ab32630.png) | ![image](https://user-images.githubusercontent.com/11011881/208426618-c8485e44-12c4-4a63-bd76-23d9839082d4.png)

**Platform information**
 - OS: Windows 11

**Additional context**
Mouse users can bring up the *Staff/Part properties* option with a single right-click on an empty region of the score, but even if the above is fixed, keyboard users will still have to select a measure first before they press the `Menu` key. Perhaps we should add this option to the context menu for all elements, so pressing `Menu` with just a single note selected is enough to expose the *Staff/Part properties* option?

**Comment by fenugrec** [comment]
Interesting, there is something about selecting a measure with the keyboard that makes a difference : if I instead select a measure with the mouse first, then press the Menu key, I get the normal context menu.

**Comment by avvvvve** [comment]
@shoogle Was looking for another issue and found this, but I couldn't reproduce. Tested in 4.4.2 on Mac, making sure to get a range selection entirely using the keyboard, then used Shift + F10. It showed all the correct options. Perhaps it's been fixed since reporting or is Windows-only? (not at my windows machine right now to check)

**Comment by fenugrec** [comment]
FWIW cannot reproduce anymore either, on 4.3.2 (archlinux still)

**Comment by cbjeukendrup** [comment]
Seems fixed, according to my own experience and the comments above
